### DISCUSSION TITLE
Add image of signature between closing and name

### DISCUSSION THREAD

**Discussion by Tjorbenn** [mcanouil/quarto-letter#19] [Q&A] [answered]
Hi,
first of all, thanks for this template!

Is there a way, that I can add an image of my signature between the closing phrase and my name at the end of the document?

Cheers,
Tjorben

**Comment by mcanouil** [comment, marked as answer]
Add it in the `name` (or in `closing`) (simple edit of the default template example).

```qmd
author:
  - name: |
      John Doe
      `\linebreak\includegraphics[width=0.25\textwidth]{signature.jpg}`{=latex}
```

<img width="625" alt="image" src="https://github.com/user-attachments/assets/c5e58e37-874a-4692-9938-994cbb5e63db">

**Reply by Tjorbenn** [reply]
Awesome thank you!

**Reply by karahosford** [reply]
This is wonderful, is there a way to add a header image such as a logo in the top left/centre of the letter?

**Reply by mcanouil** [reply]
There is almost always a way but this template is not a general one.
It serves my purpose which don't involve adding extra images, etc.
You can fork and tweak the template, or use one of the other "letter" template out there.

https://m.canouil.dev/quarto-extensions/
